docs/source/models/intervention_models/intrapartum/misoprostol_intervention.rst

```diff
@@ -0,0 +1,268 @@
+and :math:`p(\text{maternal_hemorrhage}|\text{no misoprostol})`), which we can solve analytically using the same approach as in the :ref:`cpap calibration <cpap_calibration>`.
+
+**Alternative PAF Derivation**: An alternative, and possibly simpler derivation of the PAF that will calibrate this model comes from the observation that
+:math:`\text{PAF} = 1 - \frac{1}{\mathbb{E}(\text{RR})}`.  If we define 
+
+.. math::
+
+   p(\text{no misoprostol}) = \sum_{\text{paths without misoprostol}} p(\text{path}),
+
+then can use this to expand the identity
+
+.. math::
+
+   \text{PAF}_\text{no misoprostol} = 1 - \frac{1}{\mathbb{E}(\text{RR})}.
+
+Since our risk exposure has two categories,
+
+.. math::
+
+   \mathbb{E}(\text{RR}) = p(\text{no misoprostol}) \cdot \text{RR}_\text{no misoprostol} + (1 - p(\text{no misoprostol})) \cdot 1.
+
+
+
+
+Assumptions and Limitations
+---------------------------
+
+- We assume that the relative risk of maternal hemorrhage incidence with misoprostol in practice is a value that we can find in the literature (Note: 
+  the value we are using is from [Tunçalp-2012-Cochrane-Review]_.)
+- We only consider the use of misoprostol in the prevention of maternal hemorrhage, despite other documented clinical uses of misoprostol,
+  such as for therapeutic abortion.
+- We currenty do not model the increased risk of hyperpyrexia due to misoprostol consumption, because this adverse effect is most likely to occur 
+  when dosage is higher than the recommended 600 micrograms of misoprostol. (Note: [Hofmeyr-et-al-2013-Cochrane-Review]_ found that "Pyrexia (defined as body temperature over 38°C) was increased with misoprostol compared 
+  with controls (56 studies, 2776/25,647 (10.8%) versus 614/26,800 (2.3%); average RR 3.97, 95% CI 3.13 to 5.04; Tau² = 0.47, I² = 80%). The effect 
+  was greater for trials using misoprostol 600 µg or more (27 studies; 2197/17,864 (12.3%) versus 422/18,161 (2.3%); average RR 4.64; 95% CI 3.33 to 
+  6.46; Tau² = 0.51, I² = 86%) than for those using misoprostol 400 µg or less (31 studies; 525/6751 (7.8%) versus 185/7668 (2.4%); average RR 3.07; 
+  95% CI 2.25 to 4.18; Tau² = 0.29, I² = 58%)".)
+- We assume that baseline coverage for misoprostol in home births is 5% (this is not data-backed).
+- We do not model use of misoprostol for prevention of maternal hemorrhage in BEmONC and CEmONC facilities based on the [WHO-2020]_ recommendation and with
+  the assumption that BEmONC and CEmONC facilities in our locations of interest have injectible uterotonics such as oxytocin widely available for the 
+  prevention of maternal hemorrhage. (Note: in 2020, our team did a literature review of uterotonic coverage in facility settings in LMICs, the DRC, India, 
+  Kenya, and Nigeria, and found 0-1% of facilities had misoprostol available for PPH prevention and 43-94% of facilities had oxytocin available).
```

Comment:
It would be great to have some sort of reference to this so we can look back at it, even it's just the J drive link!